compiler/rustc_builtin_macros/src/deriving/generic/ty.rs

```diff
@@ -154,7 +154,7 @@ fn mk_ty_param(
         .iter()
         .map(|b| {
             let path = b.to_path(cx, span, self_ident, self_generics);
-            cx.trait_bound(path)
+            cx.trait_bound(path, false)
```

Comment:
This prevents us from generating *methods* with const bounds on their parameters, but none of the built-in impls actually *use* type parameters on their methods, so I didn't really want to thread a boolean down 3 levels to this call site.